Backpack/Calendar/Classes/BPKCalendar.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#import "BPKCalendarAppearance.h"
+
+/**
+ * Enum values for specifying calendar selection type
+ */
+typedef NS_ENUM(NSUInteger, BPKCalendarSelection) {
+    BPKCalendarSelectionSingle = 0,
+    BPKCalendarSelectionRange = 1,
+    BPKCalendarSelectionMultiple = 2,
+};
+
+@protocol BPKCalendarDelegate
+
+/**
+ * Called when a date is selected.
+ *
+ * @param date The date selected.
+ */
+- (void)didSelectDate:(NSDate *)date;
+
+/**
+ * Called when a date is deselected.
+ *
+ * @param date The date deselected.
+ */
+- (void)didDeselectDate:(NSDate *)date;
+
+@end
+
+@interface BPKCalendar : NSObject
+
+/**
+ * Locale used for displaying name of days and months
+ */
+@property (copy, nonatomic) NSLocale *locale;
+
+/**
+ * First day of the week
+ */
+@property (assign, nonatomic) NSUInteger firstWeekday;
+
+/**
+ * Type of selection allowed
+ */
+@property (assign, nonatomic) BPKCalendarSelection selectionType;
```

Comment:
you don't need to specify `assign` and `strong`, because those are default values